tests/versioning.go

```diff
@@ -5022,6 +5024,13 @@ func (s *VersioningIntegSuite) waitForWorkflowBuildId(
 	)
 }
 
+// validateWorkflowBuildIds gets the specified workflow execution from visibility and validates that the BuildIds
```

Comment:
I don't think DescribeWorkflowExecution make visibility calls. it loads the execution, including its SAs, from the DB. so this comment is not accurate.